src/pyramid/encode.py

```diff
@@ -70,13 +70,13 @@ def urlencode(query, doseq=True, quote_via=quote_plus):
         if is_nonstr_iter(v):
             for x in v:
                 x = quote_via(x)
-                result += '%s%s=%s' % (prefix, k, x)
```

Comment:
I love everything about this PR.

But I think the changes in this function specifically end up with less readable code.

I think the sprintf (or maybe the format string style) would be more readable than the f-string style.
```
%s%s=%s
%s%s=
%s%s=%s

{}{}={}
{}{}=
{}{}={}
```

In my opinion these styles cases are easier to read at a glance than:

```
{prefix}{k}={x}
{prefix}{k}=
{prefix}{k}={v}
```

Like I said, I love this PR, but I think in this particular case it ends up with less readable code, not more readable code.

Just 2️⃣ 🪙

Reply:
I much prefer the f-string equivalent as much easier to read. It doesn't require my eyes jumping back and forth between where the variable is, and what the type is that we are formatting into the string.

Reply:
My comment was 100% bike shedding and I love this PR.

Reply:
f-strings and percent formatting are marginally faster than the `''.format()` too. f-strings ftw.